doc/api/inspector.md

```diff
@@ -52,6 +52,13 @@ parameter usage.
 
 Return the URL of the active inspector, or `undefined` if there is none.
 
+## inspector.waitForDebugger()
```

Comment:
Should This include a timeout to prevent blocking forever?

Reply:
Adding timeout for this (as well as inspector.open) is definitely something we should do. I think it should be a separate PR.

Reply:
Sounds reasonable to me. I prefer to do this work in separate PR if you do not mind.